integration_tests/packages.yml

```diff
@@ -1,2 +1,4 @@
 packages:
-  - local: ../
\ No newline at end of file
+  - local: ../
+  - package: dbt-labs/audit_helper

```

Comment:
Wrong `packages.yml` file. This needs to be in the top-level one

Reply:
Done